resources/south-america/brazil/mapeadoresIVIDES.json

```diff
@@ -0,0 +1,51 @@
+{
+  "id": "OSM-group",
+  "type": "group",
+  "locationSet": {
+  
+    "include": ["076"],
+    "include": ["005"],
+    "include": ["019"],
+    "include": ["rio_de_janeiro.geojson"]
+  
+  },
+  "languageCodes": ["pt"],
+  "order": 4,
+  "strings": {
+    "community": "Mapeadores IVIDES",
+    "comunityID": "mapeadoresIVIDES",
+    "description": "All are welcome! To see details, please, visit {signupUrl}",
```

Comment:
giving some basic info would be nice - what it is? Discussion group? Coordination for HOT mappers? TODO ideas?

(note, I am not NSI maintainer and not checked is `To see details, please, visit {signupUrl}` accepted as a good idea)

Reply:
It is publicly (and no needs authentication) in URL https://tasks.hotosm.org/organisations/instituto-virtual-para-o-desenvolvimento-sustentavel-ivides
So, I changed the {signupUrl} to {url} to provide this info. Thank you.